src/Posts/Menu.jsx

```diff
@@ -94,7 +132,14 @@ const buildMenu = (accountId, blockHeight) => {
     },
   ];
 
+  const promoteToBlogSubmenu = [];
+
   if (blockHeight) {
+    promoteToBlogSubmenu.unshift({
```

Comment:
You might also need to check the item type (path?) to see that it is /post/main since this menu also applies to comments (and other content in the future).